src/pages/lessons/[slug].tsx

```diff
@@ -195,28 +141,22 @@ const Lesson: FunctionComponent<LessonProps> = ({initialLesson}) => {
   const {authToken, logout} = useViewer()
   const [playerState, send] = useMachine(playerMachine)
 
+  console.log({playerState})
+
   const {data = {}, error} = useSWR(
-    lessonQuery,
-    lessonLoader(initialLesson.slug, authToken),
+    [initialLesson.slug, authToken],
```

Comment:
this was the refactor we did on the screen.

`useSWR` first argument is the **key** so the query was causing this to fail. We want to reload for a new lesson and/or authToken